docs/domains/Domain.json

```diff
@@ -223,18 +252,23 @@
           "description": "Experimental feature configurations.",
           "$ref": "#/definitions/Experimental"
         },
-        "opssWalletSecret": {
-          "description": "opss key passphrase.",
-          "$ref": "https://github.com/garethr/kubernetes-json-schema/blob/master/v1.13.5/_definitions.json#/definitions/io.k8s.api.core.v1.SecretReference"
-        },
         "serverService": {
           "description": "Customization affecting ClusterIP Kubernetes services for WebLogic Server instances.",
           "$ref": "#/definitions/ServerService"
         },
         "domainHome": {
-          "description": "The folder for the WebLogic Domain. Not required. Defaults to /shared/domains/domains/domainUID if domainHomeInImage is false. Defaults to /u01/oracle/user_projects/domains/ if domainHomeInImage is true.",
+          "description": "The folder for the WebLogic Domain. Not required. Defaults to /shared/domains/domains/domainUID if domainHomeSourceType is not Image. Defaults to /u01/oracle/user_projects/domains/ if domainHomeSourceType is Image.",
           "type": "string"
         },
+        "domainHomeSourceType": {
+          "description": "Domain home source type: Legal values: Image, PersistentVolume, FromModel. Defaults to Image unless configuration.model is provided and then defaults to FromModel.",
```

Comment:
I might be wrong, but I think in other places you mentioned the default it simply says 'defaults to Image'.  My preference would be to keep things simple and make the default Image regardless of whether the (optional) model config map is supplied instead of changing the doc.  Not a very strong preference, but we've been bitten by dynamic default patterns enough that I'm wary of them -- some end-users find them to be subtle, plus they IMO, they make a given domain resource less self-documenting.